src/geophires_x/AGSWellBores.py

```diff
@@ -871,12 +871,12 @@ def CalculateNonverticalPressureDrop(self, model:Model, time_operation: float, t
             # nonvertical wellbore fluid conditions based on current temperature
             rhowater = density_water_kg_per_m3(
                 self.NonverticalProducedTemperature.value[year],
-                pressure=model.reserv.lithostatic_pressure()
+                pressure=model.reserv.lithostatic_pressure(model.reserv.rhorock.value, model.reserv.Trock.value)
```

Comment:
Additional comments re: whether `lithostatic_pressure` should be taking parameters at all - `model.reserv.Trock` is temperature but the `lithostatic_pressure` is expecting depth as the second argument? (Passing pint quantities serves as a guardrail against this sort of unit confusion if the parameters are indeed mixed up here)

Reply:
Good catch—AI code-autocompletion-induced bug. Corrected in all cases. Used Pint quantities in all cases as guardrails.